packages/core/documentation/README.md

```diff
@@ -0,0 +1,39 @@
+# @kong-ui-public/documentation
+
+A set of Kong UI components for displaying documents.
+
+- [Requirements](#requirements)
+- [Components](#components)
+  - [`DocumentationContent`](#documentationcontent)
+  - [`DocumentationDisplay`](#documentationdisplay)
+  - [`DocumentationPageEmptyState`](#documentationpageemptystate)
+  - [`ProductDocumentModal`](#productdocumentmodal)
+- [Usage](#usage)
+  - [Install](#install)
+
+## Requirements
+
+- `vue` must be initialized in the host application
+- `@kong/kongponents` must be available as a `dependency` in the host application, along with the package's style imports. [See here for instructions on installing Kongponents](https://kongponents.konghq.com/#globally-install-all-kongponents).
+
+## Components
+
+Note: Each component has it's own README.md providing details about component props and events.
+
+### `DocumentationContent`
```

Comment:
Suggestion: can we have a bulletted list of clickable links instead?

Reply:
good suggestion, fixed